src/io/flutter/logging/FlutterLogTree.java

```diff
@@ -68,6 +68,65 @@ void appendStyled(FlutterLogEntry entry, String text) {
       abstract void render(FlutterLogEntry entry);
     }
 
+    class Column extends ColumnInfo<DefaultMutableTreeNode, FlutterLogEntry> {
+      boolean show = true;
+      @NotNull
+      private final TableCellRenderer renderer;
+
+      Column(@NotNull String name, @NotNull TableCellRenderer renderer) {
+        super(name);
+        this.renderer = renderer;
+      }
+
+      boolean isShowing() {
+        return show;
+      }
+
+      @Nullable
+      @Override
+      public FlutterLogEntry valueOf(DefaultMutableTreeNode node) {
+        if (node instanceof FlutterEventNode) {
+          return ((FlutterEventNode)node).entry;
+        }
+        return null;
+      }
+
+      @Override
+      public TableCellRenderer getCustomizedRenderer(DefaultMutableTreeNode o, TableCellRenderer renderer) {
+        return renderer;
```

Comment:
We should return `this. renderer`
(Because method's parameters also named renderer)

Reply:
👍